src/definitions/modules/progress.js

```diff
@@ -911,7 +986,8 @@ $.fn.progress.settings = {
     success : false,
     warning : false,
     percent : '{percent}%',
-    ratio   : '{value} of {total}'
+    ratio   : '{value} of {total}',
+    bars    : ['']
```

Comment:
New setting has to be documented in the Docs PR

Reply:
Added in https://github.com/fomantic/Fomantic-UI-Docs/pull/107/commits/e8d0febc9b6a11042e915678d85144b6b5077fe8

![image](https://user-images.githubusercontent.com/127635/54873867-35815100-4e23-11e9-8a8c-90e43ee744e0.png)